code/main/LocalSaving.ts

```diff
@@ -69,7 +69,7 @@ module LocalSaving{
     }
     
     export function supportsLocalSaving(): boolean{
-        if('localStorage' in window && window['localStorage'] !== null && 'applicationCache' in window && window['applicationCache'] !== null)
```

Comment:
So this change is enough to make save function working again?🤔

I can cherry-pick only this, if you want to

Reply:
Yes, it's enough. Please cherry-pick it.